packages/commons/src/config/fileManagerConfig.ts

```diff
@@ -1,34 +1,39 @@
 import { z } from "zod";
 
-export const FileManagerConfig = z
-  .discriminatedUnion("MOCK_FILE_MANAGER", [
+const LocalMinioConfig = z
```

Comment:
Makes sense, I can rename these configurations into a more generic `S3_LOCAL_SERVER` or similar

Reply:
I refactored this part a little bit, to make it more generic and not tied specifically to MinIO - it makes sense also because theoretically this approach can work with any S3-compatible server